handlers/resolution_resource.go

```diff
@@ -63,17 +63,55 @@ func HandleCreateResolution(svc dao.Service) http.Handler {
 			utils.WriteJSONWithStatus(w, req, m, http.StatusBadRequest)
 			return
 		}
+		// Validate if an attachment has been supplied
+		if len(resolutionDao.Attachments) == 0 {
+			log.ErrorR(req, fmt.Errorf("invalid attachment"))
+			m := models.NewMessageResponse(fmt.Sprintf("no attachment has been supplied"))
+			utils.WriteJSONWithStatus(w, req, m, http.StatusBadRequest)
+			return
+		}
+		// Validate if more than one attachment has been supplied
+		if len(resolutionDao.Attachments) > 1 {
+			log.ErrorR(req, fmt.Errorf("invalid attachments"))
+			m := models.NewMessageResponse(fmt.Sprintf("only one attachment can be supplied: %s", resolutionDao.Attachments))
+			utils.WriteJSONWithStatus(w, req, m, http.StatusBadRequest)
+			return
+		}
 
-		statusCode, err := svc.CreateResolutionResource(resolutionDao, transactionID)
-		if err != nil {
-			log.ErrorR(req, err)
-			m := models.NewMessageResponse(err.Error())
-			utils.WriteJSONWithStatus(w, req, m, statusCode)
+		isAttachmentValid := true
```

Comment:
This boolean is a bit redundant because we return and end the method with the 2 invalid checks